docs/code/StatisticsBackendTests.cpp

```diff
@@ -424,7 +424,7 @@ void dump_load_examples()
     {
         //CONF-DUMP-LOAD-EXAMPLE
         // Save the database to a file
-        StatisticsBackend::dump_database("new_backend_dump.json");
+        StatisticsBackend::dump_database("new_backend_dump.json", false);
```

Comment:
If the `clear` has `false` as default value, we can leave this as it was. I think it is clearer.